src/scripts.js

```diff
@@ -1,7 +1,88 @@
+//~~~~~~~~~~~~~~~~~~~~ IMPORTS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 import './styles.css';
 import apiCalls from './apiCalls';
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-import './assets/icon_fire_symbol_unlit.png'
+import './images/turing-logo.png';
+import './images/icon_banner_add.png';
+import './images/icon_banner_remove.png';
+import './images/icon_fire_symbol_lit.png';
+import './images/icon_fire_symbol_unlit.png';
+import RecipeRepository from '../src/classes/RecipeRepository';
+//~~~~~~~~~~~~~~~~~~~~ QUERY SELECTORS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const recipePreview = document.querySelector('.recipe-preview');
+const recipeSection = document.querySelector('.recipes-section');
 
-console.log('Hello world');
+//~~~~~~~~~~~~~~~~~~~~ GLOBAL VARIABLES ~~~~~~~~~~~~~~~~~~~~~~~
+var recipeRepository = new RecipeRepository();
+recipeRepository.addRecipes();
+var recipes = recipeRepository.recipeObjects;
+
+
+//~~~~~~~~~~~~~~~~~~~~ EVENT LISTENERS  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+window.addEventListener('load', () => {initiatePage()});
+// recipePreview.addEventListener('click', () => {viewRecipeDetail()});
+
+
+//~~~~~~~~~~~~~~~~~~~~ EVENT HANDLERS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+var initiatePage = () => {
+  createRecipePreview();
+}
+
+var createRecipePreview = () => {
+  recipeSection.innerHTML = "";
+  recipes.forEach((recipe) => {
+    recipe.showDisplayTag()
+    recipe.collectIngredients();
+    recipeSection.innerHTML += `
+      <section class="recipe-preview" id="${recipe.id}">
+        <section class="recipe-heading">
+          <h3>${recipe.name}</h3>
+        </section>
+        <div class="recipe-img">
+          <img src="${recipe.img}">
+        </div>
+        <section class="recipe-info">
+          <section class="tag-icon-section">
+            <div class="icons-section">
+              <img class="icon add-to-cook" src="./images/icon_fire_symbol_unlit.png">
+              <img class="icon add-to-saved" src="./images/icon_banner_add.png">
+            </div>
+          </section>
+        </section>
+      </section>
+      `
+  })
+}
+
+
+
+
+
+
+
+
+
+
+//~~~~~~~~~~~~~~~~~~~~ CODE/PSUEDOCODE DUMP ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
we need to remember to remove this at the end, but great addition in the interum.